sonar-dependency-check-plugin/src/main/java/org/sonar/dependencycheck/DependencyCheckConfiguration.java

```diff
@@ -48,37 +48,32 @@ public static List<PropertyDefinition> getPropertyDefinitions() {
                         .description("path to the 'dependency-check-report.html' file")
                         .defaultValue(DependencyCheckConstants.HTML_REPORT_PATH_DEFAULT)
                         .build(),
-                PropertyDefinition.builder(DependencyCheckConstants.SEVERITY_BLOCKER)
+                PropertyDefinition.builder(DependencyCheckConstants.SEVERITY_HIGH)
                         .onQualifiers(Qualifiers.PROJECT)
                         .subCategory(DependencyCheckConstants.SUB_CATEGORY_SEVERITIES)
-                        .name("Blocker")
-                        .description("Minimum score for blocker issues or -1 to deactivate blocker issues.")
-                        .defaultValue("9.0")
+                        .name("High")
+                        .description("Minimum score for high issues or -1 to deactivate high issues.")
+                        .defaultValue(Float.toString(DependencyCheckConstants.SEVERITY_HIGH_DEFAULT))
                         .type(PropertyType.FLOAT)
+                        .index(1)
                         .build(),
-                PropertyDefinition.builder(DependencyCheckConstants.SEVERITY_CRITICAL)
+                PropertyDefinition.builder(DependencyCheckConstants.SEVERITY_MEDIUM)
                         .onQualifiers(Qualifiers.PROJECT)
                         .subCategory(DependencyCheckConstants.SUB_CATEGORY_SEVERITIES)
-                        .name("Critical")
-                        .description("Minimum score for critical issues or -1 to deactivate critical issues.")
-                        .defaultValue(Float.toString(DependencyCheckConstants.SEVERITY_CRITICAL_DEFAULT))
+                        .name("Medium")
+                        .description("Minimum score for medium issues or -1 to deactivate medium issues.")
+                        .defaultValue(Float.toString(DependencyCheckConstants.SEVERITY_MEDIUM_DEFAULT))
                         .type(PropertyType.FLOAT)
+                        .index(2)
                         .build(),
-                PropertyDefinition.builder(DependencyCheckConstants.SEVERITY_MAJOR)
+                PropertyDefinition.builder(DependencyCheckConstants.SEVERITY_LOW)
```

Comment:
Please set the configuration option `deprecatedKey` to the old Minor configuration key.